routes/api.php

```diff
@@ -81,5 +81,6 @@
         Route::post('/admin_edit_question', [QuestionController::class, 'editQuestion']);
         Route::post('/admin_edit_choices', [QuestionController::class, 'editChoices']);
         Route::get('/admin/nested-categories/{subcategory_id}', [CategoryController::class, 'getSubcategoryParentCategories']);
+        Route::post('/admin/profileEdit', [ChangeNameEmailController::class, 'restore']);
```

Comment:
```suggestion
        Route::post('/admin/profile-edit', [ChangeNameEmailController::class, 'restore']);
```

please follow the other naming conventions